Add Project tomls and EKP dep for examples, (#172)
CliMA/EnsembleKalmanProcesses.jl: docs/src/installation_instructions.md

> julia --project -e 'using Pkg; Pkg.instantiate()'
```

!!! info "Do I need to clone the repository?"

>     Most times, cloning the repository in not a necessity. If you only wanna use the package's
>     functionality then merely adding the packages as a dependency on your project will do the
>     job.

Reviewer: ```suggestion
    Most times, cloning the repository in not necessary. If you only want to use the package's
    functionality, adding the packages as a dependency on your project is enough.
```

> julia --project=docs/ docs/make.jl
```

The locally rendered HTML documentation can be viewed at `docs/build/index.html`

### Running repository examples

We have a selection of examples, found within the `examples/` directory to demonstrate different use of our toolbox.
Each example directory contains a `Project.toml`

To build with the latest `EnsembleKalmanProcesses.jl` release:
````
> cd examples/example-name/
> julia --project -e 'using Pkg; Pkg.instantiate()'
> julia --project example-file-name.jl
```
If you wish to run a local modified version of `EnsembleKalmanProcesses.jl` then try the following (starting from the `EnsembleKalmanProcesses.jl` package root)

> ```

Reviewer: This is still rendered as code for some reason, maybe a missing ` somewhere.

> cd examples/example-name/
> julia --project 
> julia> ]
> (example-name)> rm EnsembleKalmanProcesses.jl
> (example-name)> dev ../..
> (example-name)> instantiate
```
followed by

> ```

Reviewer: same here.

Author: found it
